### PR TITLE
Shorten client:check and service:check output in CI 

### PR DESCRIPTION
**Pull request by jgzuke**
TODO:

- [x] Update CHANGELOG.md\* with your change (include reference to issue & this PR)
- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass

\*Make sure changelog entries note which project(s) has been affected. See older entries for examples on what this looks like.

### CODE REVIEW

**Comment by jgzuke** [comment]
Waiting for https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36924 for correct types here

**Comment by jgzuke** [comment]
@evans I put this behind a flag, we could also have this be automatic when the `CI` env variable is present, im not sure if all test runners respect that flag though

**Comment by jgzuke** [comment]
@evans https://circleci.com/gh/apollographql/apollo-tooling/28699 vs https://circleci.com/gh/apollographql/apollo-tooling/28534 for the output sifference

File: packages/apollo/src/commands/service/__tests__/check.test.ts [hunk 1]
```diff
@@ -369,6 +372,11 @@ describe("service:check", () => {
 
     nock.disableNetConnect();
 
+    delete process.env.CI;
```
Comment (evans) [inline packages/apollo/src/commands/service/__tests__/check.test.ts]: re these lines, I think this is a fine solution. Not sure there's a more effective way to simulate this environment

**Pull request merged**